flush-print/countdown_sepnewline.py

```diff
@@ -0,0 +1,6 @@
+from time import sleep
```

Comment:
Not 100% convinced by this example. Seems like a bit of a stretch and not what `sep` is for.

Reply:
Added an s for seconds and i'll update it with a note in the tutorial text as well.